cmd/gpu_plugin/rm/gpu_plugin_resource_manager.go

```diff
@@ -65,7 +65,7 @@ type podCandidate struct {
 	allocationTargetNum int
 }
 
-type deviceInfo struct {
+type DeviceInfo struct {
```

Comment:
@uniemimu PTAL. the linter error was that `NewDeviceInfo` returns unexported type. (revive/unexported-return)